cinema/urls.py

```diff
@@ -1 +1,21 @@
-# write urls here
+from rest_framework import routers
+from django.urls import include, path
+
+from cinema.views import (
+    ActorViewSet,
+    GenreViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+    CinemaHallViewSet
+)
+
+app_name = "cinema"
+
+router = routers.DefaultRouter()
+router.register("actors", ActorViewSet)
+router.register("genres", GenreViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+
+urlpatterns = [path("", include(router.urls))]
```

Comment:
When basepath is empty no need to include, instead use:
urlpatterns = router.urls